cli/src/pcluster/templates/imagebuilder_stack.py

```diff
@@ -124,26 +124,28 @@ def __init__(self, scope: core.Construct, construct_id: str, imagebuild: ImageBu
 
         imagebuilder.CfnComponent(
             self,
-            id="PClusterComponent",
-            name="-".join(["PCluster", resources_prefix]),
+            id="ParallelClusterComponent",
+            name="-".join(["ParallelCluster", resources_prefix]),
             version=utils.get_installed_version(),
             tags=build_tags,
-            description="Bake PCluster AMI",
+            description="Bake ParallelCluster AMI",
             platform="Linux",
             data=core.Fn.sub(load_yaml(imagebuilder_resources_dir, "pcluster_install.yaml")),
```

Comment:
we can rename the file to `parallelcluster.yaml`

Reply:
done